doc/source/Resources/pyaedt_installer_from_aedt.py

```diff
@@ -222,14 +228,14 @@ def install_pyaedt():
                 # Extract all contents to a directory. (You can specify a different extraction path if needed.)
                 zip_ref.extractall(unzipped_path)
             if args.version <= "231":
-                run_command('"{}" install --no-cache-dir --no-index --find-links={} pyaedt[all]=="0.9.3"'.format(pip_exe,
```

Comment:
Why are you going back to 0.9.0 ?

Reply:
I have tested 0.9.3, and some dependencies were failing. So, after some tests, the last PyAEDT release compatible with 3.7 is PyAEDT 0.9.0 (at least a complete installation using the PyAEDT installer)